packages/suite/src/views/suite/SwitchDevice/DeviceItem/DeviceItem.tsx

```diff
@@ -36,6 +36,13 @@ export const DeviceItem = ({
     isFullHeaderVisible,
 }: DeviceItemProps) => {
     const selectedDevice = useSelector(selectSelectedDevice);
+    const { getDiscoveryStatus } = useDiscovery({
+        device,
+    });
+    const discoveryStatus = getDiscoveryStatus();
+    const discoveryInProgress = Boolean(
+        discoveryStatus && discoveryStatus.status === 'loading' && device.state,
```

Comment:
disabled the adding wallets buttons when device is having its discovery running + the device state is defined (otherwie, the discovery state is loading)